version.bzl

```diff
@@ -1,3 +1,3 @@
 """The version of rules_rust."""
 
-VERSION = "0.53.0"
+VERSION = "0.54.0"
```

Comment:
Lets bump the version in another PR to give others a chance to get some changes in.

Reply:
Reverted bump